### ISSUE TITLE
Explain the intentions behind this project

### ISSUE THREAD

**Issue by Joetib** [Agent-Hellboy/mysite#1]
A little about or description about the project will help clarify its usage and purpose. And what others can do to help.

This could be in the `readme.md` of the project

**Comment by Agent-Hellboy** [comment]
Hi @Joetib
I had built this project to get familiar with the team member as we were participating in python-discord summer code jam and every member are new to each other as teams were formed randomly between qualified users.

I think I had created this repo to test something for our main project.

**Comment by Joetib** [comment]
Then may I ask the intensions of having pysimplegui in there?

**Comment by Agent-Hellboy** [comment]
I think it's to open an editor to edit code and showing diffs between 2 commits.
I think I had removed this functionality in later commits.
We were making a version control system.

**Comment by Joetib** [comment]
Oh I see
Do you still intend to keep working on the project?

**Comment by Agent-Hellboy** [comment]
I will improve the README as soon as possible.

**Comment by Agent-Hellboy** [comment]
> Oh I see
> Do you still intend to keep working on the project?

Yes why not. If you want to add new feature feel free to raise a PR. 
I am open to any kind of contribution.

**Comment by Joetib** [comment]
Great

**Comment by Joetib** [comment]
May I ask where you school? Just a side question I guess

**Comment by Agent-Hellboy** [comment]
**India.**

**Comment by Joetib** [comment]
Wow, I'm in Ghana, heading for third year
